README.md

```diff
@@ -73,7 +101,7 @@ Example existing AWS account tags:
 | 333344445555 | `Other Program / 654321` |
 
 
-#### Example Output
+#### Example Program Code Output
```

Comment:
do we need to provide additional example or update the below example?

Reply:
yeah, I can provide another example